### PR TITLE
Profile Navigation

### PR DESCRIPTION
**Pull request by Josh-Kohl**
## Description

Added file structure for Account Settings feature and navigation between Profile and Account Components. 
Also added keys to Flatlist of previous bites to address warning message. 

## How to test

Navigate to Profile page. 
Click "Personal Information" under Account Settings to navigate to placeholder Settings page.
Clicking back arrow in nav bar to direct back to Profile page.  
From Profile clicking the back arrow again directs to CurrentSession. 

## Notes

Required a bit of refactoring and restyling on the Profile component. Aesthetics are largely the same but still need to be polished after feature freeze.

### CODE REVIEW

File: src/features/ProfileView/Profile.Greeting.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const ProfileGreeting = () => {
         />
       </View>
 
-      <Text style={styles.profileName}>Hi, I'm {accountHolderName}</Text>
+      <Text style={styles.profileName}>Welcome back {accountHolderName}</Text>
```
Comment (tm2838) [inline src/features/ProfileView/Profile.Greeting.js]: lol could we use just 'Welcome' or sth else? 'Welcome back' feels not very accurate to me cuz it's not the very first screen the user sees when log back in?

**Review by tm2838 — APPROVED** [review]
lgtm! left a super minor note which doesn't really matter lol

File: src/features/JoinView/GuestQR.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ export default function QRScanner() {
       joinRequest: 'pending',
       isHost: false,
       individualBills: 0,
-      name: 'Tom',
+      name: 'Josh',
```
Comment (ashequl93) [inline src/features/JoinView/GuestQR.js]: Why are you hard-coded your name? Shouldn't I be `name: nickname || accountHolderName, //Get userName from google`

File: src/features/ProfileView/Profile.Greeting.js (hunk 1 above)
Comment (Josh-Kohl) [reply src/features/ProfileView/Profile.Greeting.js]: Sure thing! I was thinking that needed tweaking as well.

File: src/features/JoinView/GuestQR.js (hunk 2 above)
Comment (Josh-Kohl) [reply src/features/JoinView/GuestQR.js]: Ah! That was set when we were first testing the guest/host walkthrough.

**Review by ashequl93 — APPROVED** [review]
Thanks a lot, Josh. Looks good to me.

**Pull request merged**